package.json

```diff
@@ -2,26 +2,33 @@
   "name": "octopull",
   "version": "1.0.2",
   "description": "Your multi repo pull request buddy, octopull your git repos",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/alvaropinot/octopull.git"
-  },
+  "license": "MIT",
+  "main": "./lib/index.js",
+  "repository": "alvaropinot/octopull",
+  "author": "Alvaro Pinot <alvaro@neatnait.com> (http://github.com/alvaropinot)",
   "keywords": [
-    "update",
+    "repository",
+    "pull",
     "git"
   ],
-  "author": "Alvaro Pinot <alvaro@neatnait.com> (http://github.com/alvaropinot)",
-  "license": "MIT",
   "bugs": {
     "url": "https://github.com/alvaropinot/octopull/issues"
   },
-  "homepage": "https://github.com/alvaropinot/octopull#readme",
+  "scripts": {
+    "clean": "rimraf lib",
+    "dev": "npm run clean && babel src -w -s -d lib",
+    "build": "npm run clean && babel src -d lib",
+    "prepublish": "npm run build"
+  },
```

Comment:
Great idea, I've added eslint with some initial config. Let's take the config as a starting point and we can update it how we like :)